plugins/plugin-nodejs/che-plugin-nodejs-lang-shared/pom.xml

```diff
@@ -30,4 +27,4 @@
             </resource>
         </resources>
     </build>
-</project>
\ No newline at end of file
+</project>
```

Comment:
It's because of applying sortpom:sort on whole che project.
I decided to commit fixed file, because it is annoying to see this file changing in VCS after sorting.